### ISSUE TITLE
/image me photos

### ISSUE THREAD

**Issue by shurane** [seansfkelley/react-drinks#1]
Have you considered photos?

IBA world has a collection of photos with each drink: http://www.iba-world.com/index.php?option=com_content&view=article&id=88&Itemid=532

There's an Android app that also has pictures: https://play.google.com/store/apps/details?id=com.n2apps.ibacocktails

These images may not be free to use, but something along those lines would be sweet.
It would surely snazz up `react-drinks`.

**Comment by seansfkelley** [comment]
Yeah, I had pictures in an early prototype of this but they were tentatively dropped. Definitely want to bring them back in some form or another, but as you note attribution can be a problem. There's also an idea I've been kicking around to use generated illustrations rather than actual photos. Most drinks fall into a fairly narrow range, appearance-wise (well, there's maybe a dozen dimensions with anywhere from 3-20 options each which is a pretty large space, but easy to describe).
